plugin/src/main/java/org/jfrog/idea/xray/persistency/types/Severity.java

```diff
@@ -4,5 +4,59 @@
  * Created by romang on 4/16/17.
  */
 public enum Severity {
-    Normal, Unknown, Minor, Major, Critical
+    Normal("Scanned - No Issues", 0),
+    Pending("Pending Scan", 1),
+    Unknown("Unknown", 2),
+    Information("Information", 3),
+    Low("Low", 4),
+    Medium("Medium", 5),
+    High("High", 6),
+
+    @Deprecated
+    Minor("Low", 4),
+    @Deprecated
+    Major("Medium", 5),
+    @Deprecated
+    Critical("High", 6);
+
+    private String severityName;
+    private int ordinal;
+
+    Severity(String severityName, int ordinal) {
+        this.severityName = severityName;
+        this.ordinal = ordinal;
+    }
+
+    public String getSeverityName() {
+        return this.severityName;
+    }
+
+    public int getOrdinal() {
+        return this.ordinal;
+    }
+
+    public boolean isHigherThan(Severity other) {
+        return this.ordinal > other.getOrdinal();
+    }
+
+    public static Severity fromString(String inputSeverity) {
+        for (Severity severity : Severity.values()) {
+            if (severity.getSeverityName().equals(inputSeverity)) {
+                return severity;
+            }
+        }
+        // Backward compatibility
```

Comment:
Question: 
Why is this switch case for the deprecated values needed? Wouldn't the above loop cover the deprecated values as well?

Reply:
That wouldn't work because the new names of `Minor`, `Major` and `Critical` are `Low`, `Medium` and `High`.